BranchMonsterFactory/ViewControllers/MonsterCreatorViewController.m

```diff
@@ -63,16 +74,16 @@ - (void)viewDidLoad {
     self.botViewLayerThree.dataSource = self;
     [self.botViewLayerThree registerClass:[ImageCollectionViewCell class] forCellWithReuseIdentifier:@"cell"];
     
-    [self.etxtName setText:[MonsterPreferences getMonsterName]];
+    [self.etxtName setText:[self.editingMonster getMonsterName]];
     
-    UIBarButtonItem *barButton = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemDone target:self.etxtName action:@selector(resignFirstResponder)];
```

Comment:
can you remove everything that you commented out? makes for cleaner diffs. we don't usually keep code around, we can always locate it on github if we really need to